packages/dd-trace/src/plugins/index.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
   get 'https' () { return require('../../../datadog-plugin-http/src') },
   get 'ioredis' () { return require('../../../datadog-plugin-ioredis/src') },
   get 'jest-circus' () { return require('../../../datadog-plugin-jest/src') },
+  get 'manual' () { return require('../../../datadog-plugin-manual/src') },
```

Comment:
Is this really a plugin though? It's not instrumenting a module, so it seems like a hack to me. This is especially true since it also needs a fake instrumentation event to be instantiated.

Reply:
this is a different option for initialization: https://github.com/DataDog/dd-trace-js/pull/3202/commits/8bcbac88324a7bb8d81ef883be91f7478561f290

Reply:
> well I'd argue that it is instrumenting a module
Is it though? If I'm not misunderstanding, it seems like they would just emit the right events. How would it work to use this outside of CI too, especially with the generic `manual` name? The more I think about it the less I think this should be a plugin, or at least not the same kind of plugin than for instrumented modules.